lib/publiccloud/azure.pm

```diff
@@ -114,18 +111,19 @@ sub upload_img {
     my ($img_name) = $file =~ /([^\/]+)$/;
     $img_name =~ s/\.vhdfixed/.vhd/;
     my $disk_name = $img_name;
+    my $storage_account = get_var('PUBLIC_CLOUD_STORAGE_ACCOUNT', 'eisleqaopenqa');
```

Comment:
Weren't you the one that was against using a default value for `PUBLIC_CLOUD_STORAGE_ACCOUNT` here, if it is already in the terraform profile? :slightly_smiling_face:

Reply:
but then, it would not make to have the same values in tf and here or?

If it is upload specific, I would just add a parameter of `upload_img()` and pass it.

Reply:
And use a variable which is like `PUBLIC_CLOUD_IMG_UPLOAD_AZURE_STORAGE_ACCOUNT`

Reply:
> but then, it would not make to have the same values in tf and here or?

well yes and no . 

Yes it **would not** make sense to have same variable for this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/provider.pm#L392 and this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/azure.pm#L21 because from test code perspective this 2 have no relations to each other 

No it **would** make sense to have same variable because from Azure perspective we speaking about exactly same thing ( https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview )